proto/reporting/input_params/front_desk_report_manifests.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package holms.types.reporting.input_params;
+option csharp_namespace = "HOLMS.Types.Reporting.ReportParams";
+option go_package = "reporting/input_params";
+
+import "google/protobuf/timestamp.proto";
+import "primitive/pb_inclusive_opsdate_range.proto";
+import "primitive/pb_local_date.proto";
+
+enum InHouseGuestReportSorting {
+	BY_GUEST_NAME = 0;
+	BY_ROOM_NUMBER = 1;
+}
+
+message FrontDeskCalendarDateReportManifest {
+	google.protobuf.Timestamp date = 1;
```

Comment:
Two small changes here:
- These should have a nullable property filter (null means no filter, show all properties)
- You can get rid of all dates on these. I discussed this extensively with Chuck and Carmen, and they said they never generate front-desk reports for dates other than today.